omf-python/tests/test_geometry_surface.py

```diff
@@ -0,0 +1,50 @@
+import omf_python
+from os import path
+from unittest import TestCase
+
+
+class TestReader(TestCase):
```

Comment:
Probably should be named `TestGeometrySurface`

Reply:
Done